core/pom.xml

```diff
@@ -125,12 +125,17 @@
       <artifactId>jetty-servlet</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlets</artifactId>
+      <scope>compile</scope>
+    </dependency>
     <!-- Because we mark jetty as provided and shade it, its dependency
```

Comment:
You can remove the comment. You say you see just one version javax.servlet, which is good, but do you now see org.eclipse.jetty.orbit:javax.servlet in the dependency tree? I'm guessing not, but just want to make sure that's what you're looking for -- any possible copy of the servlet API from another source.